### ISSUE TITLE
it's a good idea to create a new package for sc-deployer, we should de the same for sc-compiler, maybe later in another issue/pr

### ISSUE THREAD

**Issue by AurelienFT** [massalabs/massa-sc-toolkit#73]
it's a good idea to create a new package for sc-deployer, we should de the same for sc-compiler, maybe later in another issue/pr

_Originally posted by @Thykof in https://github.com/massalabs/massa-sc-toolkit/pull/69#discussion_r1056392035_

**Comment by Thykof** [comment]
blocked by https://github.com/massalabs/massa-sc-toolkit/pull/69
